index.html

```diff
@@ -51,7 +63,8 @@ <h1>My Recent Works</h1>
     </div>
     <div class="work-section">
         <div class="story">
-            <img src="images/Img Placeholder.png" alt="portfolio-image">
+            <img src="images/Img Placeholder (2).png" class="image1" alt="portfolio-image">
+            <img src="images/Img Placeholder.png" class="image2" alt="portfolio-image">
```

Comment:
- [ ] great job adding the hover effect for a card in your work section, kindly implement the effect for all of the works card, currently only the first card has the hover effect, please fix it 🙏 

![image](https://user-images.githubusercontent.com/89282221/215266016-dc0ad5de-7314-430a-afb0-23c3f6fa9368.png)

Reply:
look at the above comments its only the first card that should have that hover effect

Reply:
- [ ] @Unleashedicon I think you miss understood the first reviewer, he said `Your first project card on the row is not having the dark opacity as requred. On a clearer note, the first card on your website is what should appear after you hover the project cards` , what he said when you hover the cards you should see the design you made for the first card (before you do the changes) , so kindly implement the hover effects for the rest of the cards

Reply:
okay thanks i have updated them and submitted them\